docs/src/howto/external-datastore.md

```diff
@@ -0,0 +1,65 @@
+# How to use an external datastore
+
+Canonical Kubernetes supports using an external datastore such as ETCD instead of the bundled in dqlite datastore. This guide walks you through configuring an external ETCD datastore.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine
+- You have an external ETCD cluster
+- You have installed the Canonical Kubernetes snap
+- You have not bootstrapped the Canonical Kubernetes cluster yet
+
+```{warning}
+The selection of the backing datastore can only be changed during the bootstrap process. There is no migration path between the bundled in dqlite and the external datastores.
+```
+
+## Adjust the bootstrap configuration
+
+To use an external datastore, a configuration file that contains the required datastore parameters needs to be provided to the bootstrap command. Create a configuration file and insert the contents below while replacing the placeholder values based on the configuration of your ETCD cluster.
+
+```yaml
+datastore: external
+datastore-url: "<etcd-member-addresses>"
+datastore-ca-crt: |
+  <etcd-root-ca-certificate>
+datastore-client-crt: |
+  <etcd-client-certificate>
+datastore-client-key: |
+  <etcd-client-key>
+```
+
+* `datastore-url` field accepts a comma seperated list of addresses (e.g. `https://10.42.254.192:2379,https://10.42.254.193:2379,https://10.42.254.194:2379`)
+* `datastore-ca-crt` field accepts a certificate for the CA in PEM format
```

Comment:
Could we give the commands to generate these?

Reply:
Yeah, I think leave etcd to a different doc :+1: